apps-rendering/src/client/article.ts

```diff
@@ -468,17 +468,11 @@ function hydrateClickToView(): void {
 }
 
 function resizeEmailSignups(): void {
-	const isIframe = (elem: Element): elem is HTMLIFrameElement =>
-		elem.tagName === 'IFRAME';
-	const emailSignupIframes = document.querySelectorAll('.js-email-signup');
+	const emailSignupIframes = document.querySelectorAll(
+		'.js-email-signup > iframe',
+	);
 	Array.from(emailSignupIframes).forEach((emailSignupIframe) => {
-		if (isIframe(emailSignupIframe)) {
-			const innerIframe =
-				emailSignupIframe.contentDocument?.querySelector('iframe');
-			if (innerIframe) {
-				innerIframe.style.width = '100%';
-			}
-		}
+		(emailSignupIframe as HTMLIFrameElement).style.width = '100%';
```

Comment:
I think if you keep the `isIframe` type predicate you can use a `filter(isIframe)` before your `forEach`. I believe TypeScript is clever enough to then narrow the type of `emailSignupIframe` to `HTMLIFrameElement` for you (which is awesome!), which means you no longer need the typecast.